app/components/GettingStartedModal/useFetchSchemas.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react';
+import { useResettableState } from '../../utils/hooks';
+import type { GettingStartedSchema } from './model';
+
+function useFetchSchemas() {
+    const [gettingStartedSchema, setGettingStartedSchema] = useState<GettingStartedSchema | null>(null);
+    const [cloudSetupSchema, setCloudSetupSchema] = useState<GettingStartedSchema | null>(null);
+    const [loading, setLoading] = useState(true);
+    const [error, setError, clearError] = useResettableState<string | null>(null);
+
+    useEffect(() => {
+        Promise.all([
+            fetch('https://repository.cloudifysource.org/cloudify/getting-started/6.4/gettingStarted.schema.json')
+                .then(response => response.json())
+                .then(response => {
+                    setGettingStartedSchema(response as GettingStartedSchema);
+                }),
+            fetch('https://repository.cloudifysource.org/cloudify/getting-started/6.4/cloudSetup.schema.json')
+                .then(response => response.json())
+                .then(response => {
+                    setCloudSetupSchema(response as GettingStartedSchema);
```

Comment:
I've double checked and it seems that type assertion is redundant here
As it's redundant, we can remove it 🎸 

```suggestion
                    setGettingStartedSchema(response);
                }),
            fetch('https://repository.cloudifysource.org/cloudify/getting-started/6.4/cloudSetup.schema.json')
                .then(response => response.json())
                .then(response => {
                    setCloudSetupSchema(response);
```

Reply:
I removed it: [570e6c3](https://github.com/cloudify-cosmo/cloudify-stage/pull/2362/commits/570e6c3fabc26b7e51407e16937b13ddb318d1a1)